packages/desktop/views/dashboard/drawers/dapp-config/components/DappCard.svelte

```diff
@@ -19,10 +20,14 @@
 >
     <div class="w-full flex flex-row justify-between items-center p-2 gap-3">
         <div class="flex flex-row gap-3 items-center overflow-hidden">
-            <img class="w-10 h-10 rounded-full" src={dapp.metadata?.icons?.[0]} alt={dapp.metadata?.name} />
+            {#if dapp.metadata?.icons?.[0]}
+                <img class="w-10 h-10 rounded-full" src={dapp.metadata.icons[0]} alt={dapp.metadata.name} />
+            {:else}
+                <Avatar icon={IconName.Link} size="lg" surface={2} />
+            {/if}
```

Comment:
I think the surface/size of icon looks unbalanced:

![image](https://github.com/bloomwalletio/bloom/assets/45079109/c1697ad8-bdcd-4133-a10a-c6a039c58ad5)

Maybe there needs some extra padding around the icon or `surface={0}`.

If you choose the latter option, there is also the problem that there is no hover effect on the Avatar component.

There is also a height difference (8px) between a DappCard that has unknown or a DappCard that has metadata + URL.